scripts/templates/webdriverio.tpl.d.ts

```diff
@@ -66,6 +84,7 @@ declare namespace WebdriverIO {
         framework?: string,
         mochaOpts?: object,
         jasmineNodeOpts?: object,
+        cucumberOpts?: CucumberOptions,
```

Comment:
Same here, move to cucumber type definition

Reply:
Same as above